src/index.html

```diff
@@ -1,13 +1,53 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+
+    <div class="product-card" data-qa="card">
+      <img
+        class="product-card__image"
+        src="images/imac.jpeg"
+        alt="iMac"
+      >
+
+      <h2 class="product-card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+      <div class="product-card__code">Product code: 195434</div>
+
+      <div class="product-review">
+        <div class="stars stars--4">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+        <div class="product-views">Reviews: 5</div>
+      </div>
+
+      <div class="product-prices">
+        <div class="product_text">Price:</div>
+        <div class="product_price">$2,199</div>
+      </div>
+
+      <a
+         data-qa="hover"
+         href="#"
+         class="card-buy"
+      >
+        <div class="card-buy__botton">BUY</div>
```

Comment:
```suggestion
         BUY
```
redundant wrapper because of this you fail the test